Source/Scene/Expression.js

```diff
@@ -1198,7 +1198,7 @@ define([
         //>>includeStart('debug', pragmas.debug);
         throw new DeveloperError('Operator "-" requires a vector or number argument. Argument is ' + left + '.');
```

Comment:
@lilleyse Shouldn't these be `RuntimeError` or has style validation already happened at this point? Assuming we change them to runtime errors, that would get rid of the elint error as well because the below returns would never happen.

Even if they are runtime errors, this code makes no sense, we know left is not a number from the above failed check but we are returning `-left` anyway?